app/js/lib/speech/synthesis.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const p = Object.freeze({
+  // Properties
+  synthesis: Symbol('synthesis'),
+  supportsSynthesis: Symbol('supportsSynthesis'),
+  voice: Symbol('voice'),
+  initialised: Symbol('initialised'),
+
+  // Methods
+  populateVoice: Symbol('populateVoice'),
+});
+
+export default class SpeechSynthesis {
+  constructor() {
+    const synthesis = window.speechSynthesis;
+
+    this[p.initialised] = false;
+    this[p.supportsSynthesis] = !!synthesis;
+
+    if (this[p.supportsSynthesis]) {
+      this[p.synthesis] = synthesis;
+      this[p.populateVoice]();
+      this[p.synthesis].onvoiceschanged = this[p.populateVoice].bind(this);
+    } else {
+      this[p.synthesis] = null;
+    }
+
+    this[p.voice] = null;
+
+    Object.seal(this);
+  }
+
+  /**
+   * Speak a text aloud.
+   *
+   * @param {string} text
+   */
+  speak(text = '') {
```

Comment:
Wonder if we should put some sort of mutual exclusion condition around this - as far as I recall, once the thing is speaking, it's async.  We might end up with garbled speech if they attempt to talk over each other.  Correct me if I've gotten the detail of this API wrong.